setup.py

```diff
@@ -37,23 +32,18 @@
          long_description=open(readme).read(),
          platforms='All',
          install_requires= require,
-         packages=['ibm_db_sa'],
+         packages=['pyodbc_sa'],
         entry_points={
          'sqlalchemy.dialects': [
-                     'db2=ibm_db_sa.ibm_db:DB2Dialect_ibm_db',
-                     'db2.ibm_db=ibm_db_sa.ibm_db:DB2Dialect_ibm_db',
-                     'db2.zxjdbc=ibm_db_sa.zxjdbc:DB2Dialect_zxjdbc',
-                     'db2.pyodbc=ibm_db_sa.pyodbc:DB2Dialect_pyodbc',
-                     'db2.zxjdbc400=ibm_db_sa.zxjdbc:AS400Dialect_zxjdbc',
-                     'db2.pyodbc400=ibm_db_sa.pyodbc:AS400Dialect_pyodbc',
+                     'db2=pyodbc_sa.pyodbc:DB2Dialect_pyodbc',
+                     'db2.pyodbc=pyodbc_sa.pyodbc:DB2Dialect_pyodbc',
+                     'db2.pyodbc400=pyodbc_sa.pyodbc:AS400Dialect_pyodbc',
 
-                     # older "ibm_db_sa://" style for backwards
+                     # older "pyodbc_sa://" style for backwards
                      # compatibility
-                     'ibm_db_sa=ibm_db_sa.ibm_db:DB2Dialect_ibm_db',
-                     'ibm_db_sa.zxjdbc=ibm_db_sa.zxjdbc:DB2Dialect_zxjdbc',
-                     'ibm_db_sa.pyodbc=ibm_db_sa.pyodbc:DB2Dialect_pyodbc',
-                     'ibm_db_sa.zxjdbc400=ibm_db_sa.zxjdbc:AS400Dialect_zxjdbc',
-                     'ibm_db_sa.pyodbc400=ibm_db_sa.pyodbc:AS400Dialect_pyodbc',
+                     'pyodbc_sa=pyodbc_sa.pyodbc:DB2Dialect_pyodbc',
+                     'pyodbc_sa.pyodbc=pyodbc_sa.pyodbc:DB2Dialect_pyodbc',
+                     'pyodbc_sa.pyodbc400=pyodbc_sa.pyodbc:AS400Dialect_pyodbc'
```

Comment:
These we can just get rid of. We don't have any backward compatibility to deal with.